client/data/settings/actions.js

```diff
@@ -281,3 +282,11 @@ export function* submitStripeBillingSubscriptionMigration() {
 		'scheduleStripeBillingMigration'
 	);
 }
+
+export function saveOption( optionKey, value ) {
+	directApiFetch( {
+		path: `${ NAMESPACE }/settings/${ optionKey }`,
+		method: 'post',
+		data: { value },
+	} ).catch( () => {} );
```

Comment:
Does this empty catch mimic `updateOptions` from `wc/admin/options`?

I'd at least `console.error()` here.

Note: I realize that this might be PoC code.

Reply:
Addressed 0d5d493344c59ede904847c0dd92a0f7aad16388